test/VaultSharp.UnitTests/V1/AuthMethods/Custom/CustomAuthMethodLoginProviderTests.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using VaultSharp.Core;
+using VaultSharp.V1.AuthMethods.Custom;
+using VaultSharp.V1.Commons;
+using Xunit;
+
+namespace VaultSharp.UnitTests.V1.AuthMethods.Custom
+{
+    public class CustomAuthMethodLoginProviderTests
```

Comment:
Not sure about this.
I have noticed that there are no unit tests in the repository and decided to introduce them for current and further implementation.
If you don't feel comfortable having them in repository I can remove it from the repository.